portfolio/src/main/webapp/script.js

```diff
@@ -20,8 +20,8 @@ function addRandomGreeting() {
       ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
 
   // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
+//   const greeting = greetings[Math.floor(Math.random() * greetings.length)];
```

Comment:
nitpick: Avoid leaving commented out code in the commit (the beauty of git is that we can always go find this line of code if we want to put it back later).